### PR TITLE
Pass golint on all packages

### PR DESCRIPTION
**Pull request by dmathieu**
And remove the `assets` package, as it only holds a logo, which makes sense to be at the top of the project.

### CODE REVIEW

File: broker/io.go [hunk 1]
```diff
@@ -13,8 +13,12 @@ type writer struct {
 	channel channel
 }
 
-var ErrNotRegistered = errors.New("Channel is not registered.")
+// known errors
```
Comment (tt) [inline broker/io.go]: Is this required?

Comment (tt) [inline broker/io.go]: I see you responded to that on the other pull request. I think this is really annoying. The comment is not adding any value; it's just making the file a line longer.

File: util/util.go [hunk 2]
```diff
@@ -22,40 +22,17 @@ func NewUUID() (string, error) {
...
 }
 
-type NullByte []byte
-
-func GetNullByte() []byte {
-	return new(NullByte).Get()
-}
-
-func (nb NullByte) Get() []byte {
-	nb = []byte{0}
```
Comment (tt) [inline util/util.go]: Why was this not initialized with the type, @naaman?

**Comment by tt** [comment]
Looks good to me.

File: broker/io.go (hunk 1 above)
Comment (dmathieu) [inline broker/io.go]: I agree. But golint is an overall quite useful tool, and ignoring some warnings is really boring.

File: util/util.go (hunk 2 above)
Comment (naaman) [inline util/util.go]: 👻

**Pull request merged**